bloom-housing/bloom
# 1711/403 page

# Pull Request Template

## Issue
Closes: #1711

Addresses # (issue)


## Description

Adds a new /unauthorized page and redirects on 403 error.

## Type of change


## How Can This Be Tested/Reviewed?

Log in as non-admin, try to open the `/users` page. you should be redirected to the 403 page.


## Checklist:
